src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -283,84 +270,6 @@
   <data name="security_ServiceNameCollection_EmptyServiceName" xml:space="preserve">
     <value>A service name must not be null or empty.</value>
   </data>
-  <data name="net_allocate_ssl_context_failed" xml:space="preserve">
-    <value>Failed to allocate SSL/TLS context, OpenSSL error - {0}.</value>
-  </data>
```

Comment:
Aren't lots of these used by the Unix build?

Reply:
Yes, they are. But I do not see it in my build - I do not have unix platform. So I always wait for builds herem, check the failing logs and fix it.

Reply:
> But I do not see it in my build - I do not have unix platform

You don't need it just to build; you can build everything on Windows, e.g. to build the Linux build of the assembly, do `msbuild /p:OSGroup=Linux` from the src directory.

Reply:
Great. Thanks a lot.